src/api/stillings-api/hentKommuner.ts

```diff
@@ -34,30 +34,410 @@ export const useHentKommuner = () => {
 };
 
 const kommuneMock: KommuneDTO[] = [
```

Comment:
Trenger vel ikke så stor kommune mock?

Isåfall burde vi vel ha det som en json fil?